.github/workflows/ci.yml

```diff
@@ -47,6 +47,7 @@ jobs:
           SKIP_PYTHON_BINDINGS_TESTS: ${{ matrix.SKIP_PYTHON_BINDINGS_TESTS }}
 
   bazel:
+    if: false # doesn't work in the forked repository
```

Comment:
Pridali tam nejaky novy testovaci nastroj, https://bazel.build,
ktory ale nenfuguje v nasom forknutom repe, kedze muse nesedi github repo.

Asi by sa to dalo nejak nastavit, ale neprisiel som na to ako.
Bohato nam stacia pure Go testy.